code/items/equipments/entities/C4Entity.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using System.Linq;
```

Comment:
Linter

Reply:
It's PR into Fires branch, not master. I figured we would just do the linter thing before the master branch PR.